.ci/install_linux_catkin.sh

```diff
@@ -6,29 +6,14 @@ cd "${HOME}/workspace"
 cp -r "${GITHUB_WORKSPACE}" src
 ./scripts/internal-distro.py --workspace=src distribution.yml --repository "${REPOSITORY}" ${REQUIRED_ONLY}
 
-if [ $BUILD_NAME = TRUSTY_FULL_DEBUG ]; then
-  sudo apt-get install -y clang-format-6.0
-fi
+# Catkin on Focal-only
+$SUDO apt-get install -y clang-format-10
```

Comment:
Do we want to keep the if statements to support earlier versions of Ubuntu, or do we want to only support Focal? (Either way is fine, I just figured that documenting the preference and reasoning in this PR would be useful.)

Reply:
Let it be documented both here, in the fact that our workflows run on the ubuntu-20.04 Github-Actions machine, and the new README (which changed `Trusty` -> `Focal`), that we are indeed primarily supporting Focal (with deprecated support for Bionic until that goes end-of-life).

We can debate this in #aikido if needed, but I feel like we should generally try building to the current Ubuntu LTS and leave build support for the previous Ubuntu LTS until they go end-of-life.